src/ignoreButRetain.js

```diff
@@ -163,6 +161,7 @@ define([
         },
         parseBindElement: function (form, el, path) {
             if (this.data.ignore.active) {
+                path = xpath.parser.parse(path).toXPath();
```

Comment:
Double checked this and it would never actually get here because https://github.com/dimagi/Vellum/blob/master/src/parser.js#L599 would have failed to load the form already